src/services/ui/src/components/Opensearch/main/Filtering/FilterDrawer.tsx

```diff
@@ -16,23 +16,14 @@ import {
 
 import { FilterableSelect } from "./FilterableSelect";
 import { FilterableDateRange } from "./FilterableDateRange";
-import { FilterableCheckbox } from "./FilterableCheckbox";
+import { FilterableMultiCheck } from "./FilterableMultiCheck";
 import { useFilterDrawer } from "./useFilterDrawer";
 import { Button } from "@/components/Inputs";
-import { checkMultiFilter } from "@/components/Opensearch";
-import { useOsUrl } from "@/components/Opensearch/main";
+import { FilterableBoolean } from "./FilterableBoolean";
 
 export const OsFilterDrawer = () => {
   const hook = useFilterDrawer();
-  const url = useOsUrl();
 
-  const filtersApplied = checkMultiFilter(url.state.filters, 1);
-  const handleFilterReset = () =>
-    url.onSet((s) => ({
-      ...s,
-      filters: [],
-      pagination: { ...s.pagination, number: 0 },
-    }));
```

Comment:
Moved these into the code-behind hook.  In the future, I will push for state logic to be isolated from the JSX